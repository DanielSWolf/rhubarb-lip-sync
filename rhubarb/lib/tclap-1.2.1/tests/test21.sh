#!/bin/sh

# success
../examples/test5 -b asdf -c fdas -g asdf -j homer > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test21.out; then
	exit 0
else
	exit 1
fi

