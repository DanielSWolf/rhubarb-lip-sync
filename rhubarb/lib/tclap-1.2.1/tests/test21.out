for J:
 0  homer
for A OR B we got : asdf
for string C we got : fdas
for string D we got : 0
for E or F or G we got: asdf
